setup.py

```diff
@@ -1,11 +1,34 @@
-#!/usr/bin/env python
+from setuptools import setup
```

Comment:
Plugins don't need to be converted to pure python to work correctly on Windows. I'm doing new plugins this way because they build so much faster.

The only thing that doesn't work on Windows is ```ros2 run plugin_name plugin_name```. For ```rqt_gui``` we absolutely needed a way to do that, which is why we came up with the split rqt/rqt_gui solution.